src/ORM/Association/Loader/SelectLoader.php

```diff
@@ -194,6 +201,33 @@ protected function _buildQuery($options)
     }
 
     /**
+     * Helper method to infer the requested finder and its options.
+     *
+     * Returns the inferred options from the finder $type.
+     *
+     * ### Examples:
+     *
+     * The following will call the finder 'translations' with the value of the finder as its options:
+     * $query->contain(['Comments' => ['finder' => ['translations']]]);
+     * $query->contain(['Comments' => ['finder' => ['translations' => []]]]);
+     * $query->contain(['Comments' => ['finder' => ['translations' => ['locales' => ['en_US']]]]]);
+     *
+     * @param string|array $finderData The finder name or an array having the name as key
+     * and options as value.
+     * @return array
+     */
+    protected function _extractFinder($finderData)
```

Comment:
Is it possible to support multiple finders, eg `['finder' => ['translations' => [], 'foo', 'bar']`?

Is is supported anywhere else?